spec-tests/src/test/java/org/candlepin/spec/content/ContentVersioningSpecTest.java

```diff
@@ -0,0 +1,314 @@
+        ContentDTO expectedContent2 = new ContentDTO()
+            .id(expectedContent1.getId())
+            .name(expectedContent1.getName() + "-2")
+            .label(expectedContent1.getLabel())
+            .type(expectedContent1.getType())
+            .vendor(expectedContent1.getVendor())
+            .contentUrl(expectedContent1.getContentUrl());
+
+        ContentDTO owner1Content = adminClient.ownerContent()
+            .createContent(owner.getKey(), expectedContent1);
+        ContentDTO owner2Content = adminClient.ownerContent()
+            .createContent(owner2.getKey(), expectedContent2);
+
+        assertThat(owner1Content.getUuid()).isNotNull();
+        assertThat(owner2Content.getUuid()).isNotNull();
+        assertNotEquals(owner1Content.getUuid(), owner2Content.getUuid());
+    }
+
+    @Test
+    public void shouldCreateANewContentInstanceWhenAnOrgUpdatesASharedInstance() throws Exception {
+        OwnerDTO owner2 = adminClient.owners().createOwner(Owners.random());
+        OwnerDTO owner3 = adminClient.owners().createOwner(Owners.random());
+        ContentDTO expectedContent = Content.random();
+
+        ContentDTO owner1Content = adminClient.ownerContent().createContent(owner.getKey(), expectedContent);
+        ContentDTO owner2Content = adminClient.ownerContent().createContent(owner2.getKey(), expectedContent);
+        ContentDTO owner3Content = adminClient.ownerContent().createContent(owner3.getKey(), expectedContent);
+
+        assertThat(owner1Content.getUuid())
+            .isEqualTo(owner2Content.getUuid())
+            .isEqualTo(owner3Content.getUuid());
+
+        expectedContent.name(StringUtil.random("updated-name-"));
+        ContentDTO owner2UpdatedContent = adminClient.ownerContent()
+            .createContent(owner2.getKey(), expectedContent);
+
+        assertThat(adminClient.ownerContent().listOwnerContent(owner2.getKey()))
+            .singleElement()
+            .returns(owner2UpdatedContent.getUuid(), ContentDTO::getUuid)
+            .doesNotReturn(owner2Content.getUuid(), ContentDTO::getUuid);
+
+        assertThat(adminClient.ownerContent().listOwnerContent(owner.getKey()))
+            .singleElement()
+            .returns(owner1Content.getUuid(), ContentDTO::getUuid);
+
+        assertThat(adminClient.ownerContent().listOwnerContent(owner3.getKey()))
+            .singleElement()
+            .returns(owner3Content.getUuid(), ContentDTO::getUuid);
+    }
+
+    @Test
+    public void shouldCreateANewInstanceWhenMakingChangesToAnExistingInstance() throws Exception {
+        ContentDTO content = adminClient.ownerContent().createContent(owner.getKey(), Content.random());
+
+        content.setName(content.getName() + "-2");
+        ContentDTO updatedContent = adminClient.ownerContent()
+            .updateContent(owner.getKey(), content.getId(), content);
+
+        assertThat(updatedContent)
+            .isNotNull()
+            .extracting(ContentDTO::getUuid)
```

Comment:
This could be a `.doesNotReturn` check instead

Reply:
Updated